catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswQueryFactory.java

```diff
@@ -180,21 +174,11 @@ private CswRecordMapperFilterVisitor buildFilter(QueryConstraintType constraint,
                     CswConstants.NO_APPLICABLE_CODE,
                     null);
         }
-        visitor.setVisitedFilter(filter);
-        boolean isTypeNameCsw = typeNames.contains(new QName(CswConstants.CSW_OUTPUT_SCHEMA,
-                CswConstants.CSW_RECORD_LOCAL_NAME,
-                CswConstants.CSW_NAMESPACE_PREFIX));
-        boolean isTypeNameGmd = typeNames.contains(new QName(GmdMetacardType.GMD_NAMESPACE,
-                GmdMetacardType.GMD_LOCAL_NAME));
-
-        if (isTypeNameCsw || isTypeNameGmd) {
-            try {
-                visitor.setVisitedFilter((Filter) filter.accept(visitor, null));
-            } catch (UnsupportedOperationException ose) {
-                throw new CswException(ose.getMessage(),
-                        CswConstants.INVALID_PARAMETER_VALUE,
-                        null);
-            }
+
+        try {
+            visitor.setVisitedFilter((Filter) filter.accept(visitor, null));
+        } catch (UnsupportedOperationException ose) {
+            throw new CswException(ose.getMessage(), CswConstants.INVALID_PARAMETER_VALUE, null);
```

Comment:
Should we be logging the original exception or passing it along as throwable since the only thing that will survive of the original exception is the message?